.drone.yml

```diff
@@ -48,9 +48,6 @@ steps:
   - name: docker
     path: /var/run/docker.sock
   when:
-    instance:
-      include:
-      - drone-publish.rancher.io
```

Comment:
Are we absolutely sure we want to do this?

Reply:
I'm actually for this, so that we don't keep accidentally merging things that fail to build on publish.

It does slow down PR-build time, but the way the drone runners are set up, a drone runner is going to get provisioned for the specific pipeline even just doing a clone

Reply:
https://github.com/rancher/rancher/pull/35457

When I timed it last, it was around ten extra minutes per drone run to have both sets of provisioning tests run on PR.